packages/checkout-storefront/src/sections/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,12 @@
+import { SaleorLogo } from "@/checkout-storefront/images";
+import { getSvgSrc } from "@/checkout-storefront/lib/svgSrc";
+import { LanguageSelect } from "@/checkout-storefront/sections/PageHeader/LanguageSelect";
+
+export const PageHeader = () => {
+  return (
+    <div className="page-header">
+      <img src={getSvgSrc(SaleorLogo)} alt="logo" className="logo" />
```

Comment:
```suggestion
      <img src={getSvgSrc(SaleorLogo)} alt="" className="logo" />
```

I don't think this one needs an alt at all - if we give it an empty string it will be skipped in a11y tree